go.mod

```diff
@@ -5,5 +5,6 @@ go 1.15
 require (
 	github.com/herumi/bls-eth-go-binary v0.0.0-20201019012252-4b463a10c225
 	github.com/holiman/uint256 v1.1.1
-	github.com/kilic/bls12-381 v0.0.0-20201226121925-69dacb279461
+	github.com/kilic/bls12-381 v0.0.0-20210111123357-c0f46a7d5c1f
```

Comment:
Thanks, forgot to keep the deps updated. Might be better for another PR though. Do you plan to tag releases of your library? Or is there a changelog somewhere?

Reply:
This PR depends on very recent changes of the base library. [see](https://github.com/kilic/bls12-381/commit/c0f46a7d5c1fe4e1b185e6e6a0e3fe2041669607)

>  Do you plan to tag releases of your library?

Yes in two weeks just after a couple of improvements.